src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Windows.cs

```diff
@@ -443,9 +425,12 @@ private static ProcessInfo[] GetProcessInfos(PerformanceCounterLib library, int
                                 // at the end.  If instanceName ends in ".", ".e", or ".ex" we remove it.
                                 if (instanceName.Length == 15)
                                 {
-                                    if (instanceName.EndsWith(".", StringComparison.Ordinal)) instanceName = instanceName.Slice(0, 14);
-                                    else if (instanceName.EndsWith(".e", StringComparison.Ordinal)) instanceName = instanceName.Slice(0, 13);
-                                    else if (instanceName.EndsWith(".ex", StringComparison.Ordinal)) instanceName = instanceName.Slice(0, 12);
+                                    if (instanceName.EndsWith(".", StringComparison.Ordinal))
```

Comment:
Can this use the char-based overload?
```suggestion
                                    if (instanceName.EndsWith('.'))
```

Reply:
`instanceName` is a `ReadOnlySpan<T>`, and there isn't an overload like `ReadOnlySpan<T>.EndsWith(T)`.

Reply:
But this could use the generic version without `StringComparison`. It's a little cheaper.